client/my-sites/plugins/plugin-management-v2/index.tsx

```diff
@@ -64,30 +65,23 @@ export default function PluginManagementV2( {
 		return <div className="plugin-management-v2__no-sites">{ emptyStateMessage }</div>;
 	}
 
+	const title = translate( 'Installed Plugins' );
+
+	const rowFormatter = ( props: PluginRowFormatterArgs ) => {
+		return <PluginRowFormatter { ...props } selectedSite={ selectedSite } />;
+	};
```

Comment:
I can see we have the `rowFormatter` declared here, and then we just pass it down to the `PluginsList` component. Can we move that function to the `PluginsList` component itself to avoid passing it here? If that would require passing any extra props, it's okay to leave it as-is.